SmartDeviceLink/SDLMutableDataQueue.h

```diff
@@ -0,0 +1,35 @@
+//
+//  SDLMutableDataQueue.h
+//  
+//
+//  Created by David Switzer on 3/2/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface SDLMutableDataQueue : NSObject
+
+- (instancetype)init;
```

Comment:
Is it a convention in the SDL iOS project to not specify the prototype for init in the public interface?